server/session/event_log.go

```
@@ -103,7 +102,7 @@ func (s *Store) appendEventsLogLocked(events []Event) (int64, error) {
 	if err != nil {
 		return 0, fmt.Errorf("open events file for append: %w", err)
 	}
-	defer func() { _ = fp.Close() }()
+	defer fp.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `fp.Close` is not checked

(errcheck)